pyproject.toml

```diff
@@ -63,6 +63,9 @@ test = [
     "pytest-cov",
     "types-requests",
 ]
+uv = [
+  "build[uv]~=1.2.0",
+]
```

Comment:
Side note; I remember that `pypa/build` 1.2.0 broke things for us last year. Would it be worth it for us to vendor `pypa/build`, so that `pypabuild.py` can use it instead?

Reply:
> Side note; I remember that pypa/build 1.2.0 broke things for us last year.

I don't remember this. Could you please elaborate? Note that `build~=1.2.0` is also set in the dependencies.

> Would it be worth it for us to vendor pypa/build, so that pypabuild.py can use it instead?

No, I don't think vendoring is a good idea for `pypa/build` case. If we want to pin into a specific version, I would choose to pin the version in pyproject.toml instead.

Reply:
Ah, sounds good to me, thanks! It was this issue: https://github.com/numpy/numpy/issues/26164